index.bs

```diff
@@ -2634,8 +2637,17 @@ Multiple [=constructor operations=] may appear on a given [=interface=].
 For each [=constructor operation=] on the [=interface=], there will be a way to attempt to
 construct an instance by passing the specified arguments.
 
-The prose definition of a [=constructor operation=] must either initialize the value passed as
-<b>[=this=]</b>, or throw an exception.
+The constructor description of a [=constructor operation=] must either initialize the value passed
+as [=this=], or throw an exception.
+
+The <dfn>constructor steps</dfn> of a constructor _constructor_ should be introduced using text of
+the form “The <code>new <var>constructor</var>()</code> constructor steps are:” followed by a list
+or “The <code>new <var>constructor</var>()</code> constructor steps are to” followed by an inline
+description.
+
+<p class=example id=example-constructor-description>If the constructor does not initialize
+[=this=], one can write “The <code>new Example(<var ignore>init</var>)</code> constructor
+steps are to do nothing.”
```

Comment:
Isn't this contradictory to the above

> The constructor description of a constructor operation must either initialize the value passed
as this, or throw an exception.

Reply:
Hopefully fixed.